schedule.json

```diff
@@ -33,7 +33,7 @@
   },
   "v8": {
     "start": "2017-04-30",
-    "lts": "2016-10-31",
+    "lts": "2017-10-31",
```

Comment:
+1 for emitting the day. Feels too specific, especially since LTS does not guarantee the LTS version will be cut on that day.